runtime/doc/eval.txt

```diff
@@ -1815,7 +1815,7 @@ getcmdpos()			Number	return cursor position in command-line
 getcmdtype()			String	return current command-line type
 getcmdwintype()			String	return current command-line window type
 getcurpos()			List	position of the cursor
-getcwd()			String	the current working directory
+getcwd( [{scope}])			String	the current working directory
```

Comment:
@justinmk In many cases (not here) this is not about consistency: when doc uses conceal feature to change number of characters displayed one can align with tabs, but never with spaces: there will always be `&conceallevel` user option value which will make help file look misaligned. Not applicable for indentation.